nixos/modules/programs/lazygit.nix

```
@@ -0,0 +1,35 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.lazygit;
+
+  settingsFormat = pkgs.formats.yaml { };
+in
+{
+  options.programs.lazygit = {
+    enable = lib.mkEnableOption (lib.mdDoc "simple terminal UI for git commands");
```

```suggestion
    enable = lib.mkEnableOption "lazygit, a simple terminal UI for git commands";
```

`lib.mdDoc` is no longer necessary and they are already doing treewide removal on this.